### ISSUE TITLE
Order account users alphabetically

### ISSUE THREAD

**Issue by nokome** [stencila/hub#554]
Account users should appear alphabetically by `username`. Currently they seems to be ordered by user `id`. Also, I noticed that when changing the role of a user the order can change - so ordering alphabetically would prevent this.

![image](https://user-images.githubusercontent.com/1152336/87408822-25911000-c618-11ea-976b-3625b820fd1b.png)

**Comment by jlbrewe** [comment]
Hi! I have a project for a software methodology class to contribute to an open source repo and I am very interested in this project. Could I be assigned to this issue?

**Comment by nokome** [comment]
Hi @jlbrewe 👋🏽 , thanks for offering to take this issue on, I have assigned it to you. Please don't hesitate to let us know if you need a hand getting started.

**Comment by jlbrewe** [comment]
Thanks for the quick response! In regard to getting started, I was wondering what file(s) I should direct my attention to in order to fix this issue?

**Comment by nokome** [comment]
I think the line that needs to be changed is:

https://github.com/stencila/hub/blob/46308bb645101c3a1c8450171975cc3a83b6f2ab/manager/accounts/api/views.py#L351

To sort by the user's username you could use the Django ORM e.g.:

```python
return AccountUser.objects.filter(account=account).order_by('user__username')
```

but ideally we want to have users sorted by their displayed name (there is a function for that https://github.com/stencila/hub/blob/303416adc2150bc3b23ba21cb87554a298b4f202/manager/users/models.py#L66) so it probably needs to be done using Python `sort` on the list of `AccountUser` objects.

See the main `README.md` for instructions on getting started with a dev setup. You should only need to have the `manager` running (i.e the Django application). So something like `make -C manager create-devdb-sqlite` and then `make -C manager run` should be sufficient to be able to test the changes.

**Comment by jlbrewe** [comment]
Hi!
I am having some trouble with testing the 'get_queryset' function on line 345. When I go to the 'Users' tab of an Organization, I notice it is calling the get_queryset function on line 85. This may an issue on my end in terms of being in the right place to test this. I attempted to make changes to the function at line 85 (which is using queryset) by ordering by '-display_name', but was not able to get that working either. If you have any suggestions, that would be greatly appreciated. 

Context: 
- I have installed everything needed for dev env and have gotten it up and running
- I have ran the suggest commands mentioned above
- Everything seems to be responding live so I am assuming my dev env is set up correctly, but could be wrong.

**Comment by nokome** [comment]
Hi @jlbrewe. Great to hear that you've managed to get an dev env up and running.

Yes, you a right the `AccountsUsersViewSet.get_queryset` method was not getting called on the initial template render. This was due to the `accounts.users.all` in the template itself. I've made some modifications [here](f8824f330b4b1883dcfb2cc893ad6a021a38b3a6) so that things work as originally expected (and more consistent with the rest of the code base).

After make those changes I tested that adding `.order_by('user__username')` indeed works now. Note that using  `.order_by('display_name')` will not work because there is no such field in the database. That is what I meant by "probably needs to be done using Python sort on the list of AccountUser objects" i.e:

```python
        return sorted(
            AccountUser.objects.filter(account=account).select_related('user__personal_account'),
            key=lambda account_user: get_name(account_user.user),
        )
```

This sorts the users by their display name (e.g. "Dr Anna Anderson"). It uses `select_related('user__personal_account')` to reduce the number of database queries by "pre-selecting" the data needed.
